apps/backoffice-v2/src/pages/Entity/components/Case/Case.Actions.tsx

```diff
@@ -105,7 +105,7 @@ export const Actions: FunctionComponent<IActionsProps> = ({
               className={ctw({
                 loading: debouncedIsLoadingRejectEntity,
               })}
-              disabled={isLoading || !canReject}
+              // disabled={isLoading || !canReject}
```

Comment:
Make sure to uncomment the `disabled` lines once conflicts are resolved with dev.